en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md

```diff
@@ -0,0 +1,131 @@
+  proactively. Following sections will discuss how to create, modify and remove latency alerts.
+  
+#### Create Latency Alert
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
+- This will take you to **Latency Alert Configuration** section by default.
+
+  ![latency alert config]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+- First part of the UI has the alert creation controls and later half will showcase existing alerts and modify/remove
+ controls.
+- To configure an alert first make sure you are in the correct 
+[organization]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations) and selected
+ the desired [environment]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction). 
+- In the API name text box start typing the API you want to create alerts. Dropdown will appear with APIs which match
+ the text you have entered. Select the desired API from there. Please note that you have to invoke the API at least
+  once for the API name to be populated here. Else you will have to provide the name in the desired format as
+   instructed in the UI.
+   
+   ![api name dropdown]({{base_path}}/assets/img/observe/alerts/api-name-dropdown.png)
+- In the next column you can select the desired metric against which you evaluate the alert rule. Dropdown will show
+ all available options. If there are multiple options then you can select the desired metric. If there is only one
+  selection dropdown will be disabled.
+  
+  ![latency metric type]({{base_path}}/assets/img/observe/alerts/latency-metric-type.png)
+- In the next column you can specify the threshold in milliseconds. When the 95th percentile of the selected metric
+ exceed the threshold provided here alerts will be triggered.
+ 
+  ![latency threshold]({{base_path}}/assets/img/observe/alerts/latency-threshold.png)
+- Once all required information is added you can add the alert configuration using **Add** button. 
+- When added successfully it will be shown in the lower table.
+  
+  ![latency success]({{base_path}}/assets/img/observe/alerts/latency-success.png)
+  
+#### Modify Latency Alerts
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
+- This will take you to **Latency Alert Configuration** section by default.
+  
+  ![latency alert config]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+- Later table shows the already configured alerts. From the table select the alert you want to modify and click the edit
+ icon under **Action** column. It will make that row editable.
+ 
+  ![latency config modify]({{base_path}}/assets/img/observe/alerts/latency-config-modify.png)
+- Change the parameters as you desire and click submit icon. If you want to discard the changes click on the stop
+/remove icon.
+
+#### Remove Latency Alerts
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```

Comment:
To remove an existing latency alert:

1.	Click **User Profile** on the upper-right corner and select **Settings**.

	This takes you to the **Latency Alert Configuration** section by default. The latter table shows the already configured alerts. 

	![latency alert config]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)

2.	Select the alert you want to remove from the table and click the **bint** option under **Action**. 

Your alert configuration is now removed.